src/handlers/graphical.rs

```diff
@@ -688,6 +697,30 @@ impl GraphicalReportHandler {
         Ok(())
     }
 
+    fn render_highlighted_line_text(
+        &self,
+        f: &mut impl fmt::Write,
+        styled_text: &[Styled<&str>],
```

Comment:
> I'd be in favor of just changing `render_line_text` to take `&[Styled<&str>]`, and passing `&[line.text.styled(Style::default())]` in the unhighlighted case. Fewer different paths makes less room for bugs :)

Yeah I didn't like this approach. My initial thought was to pass in the `&mut HighlightState` but that also seemed messy. I didn't consider that `Styled` can represent a blank style as well. With that in mind, I could just make a `DefaultHighlighter` instead of using an `Option`, and eliminate all branches here.

Reply:
I also just noticed `owo_colors::StyledList` which can potentially cut down on unnecessary escape sequences, so I might change the trait to use that instead of a Vec

Reply:
> With that in mind, I could just make a DefaultHighlighter instead of using an Option, and eliminate all branches here.

Oh, this is a great idea!